lib/15_in_words.rb

```diff
@@ -0,0 +1,75 @@
+class Fixnum
+  def in_words
+    hash = {
```

Comment:
Constants like these best go outside of the method in the class. For example: 

```
class Fixnum
  HASH = { ... }

  def in_words
    ...
  end
end
```

Methods should be kept clean of large constants wherever possible.